### ISSUE TITLE
Documentation for Feature/ResourceResolver.map(..) Transformer

### ISSUE THREAD

**Issue by justinedelson** [Adobe-Consulting-Services/adobe-consulting-services.github.io#45]
Documentation for https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/564

**Comment by henrykuijpers** [comment]
We should at least specify in this documentation that it is there for the scenarios that Linkchecker doesn't cover (URLs that do not end with .html or .htm or other element types). See info in https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/564

**Comment by davidjgonzalez** [comment]
Good idea; iirc someone on gitter a few mmonths had said they could config the ootb link checker to the same effect (mapping unusual attributes/attr values) .. but maybe there were just targeting something supported? thanks for verifying

**Comment by henrykuijpers** [comment]
The guy on Gitter; That was me. :)

And I can now clarify that at least img:src is *not* working with the Linkchecker.
I believe the link checker is really limited to only a/area/form.

**Comment by davidjgonzalez** [comment]
@henrykuijpers haha - sorry - hard to remember names across gitter, disqus and gh :) ... and thanks for verifying :)
